src/__stories__/credit-card-expiration-field-story.tsx

```diff
@@ -0,0 +1,132 @@
+                            autoComplete="off"
+                            data-testid="credit-card-expiration-field"
+                            {...rest}
+                        />
+                    </div>
+                    <Stack space={8}>
+                        <Text1 regular>
+                            onChange:{' '}
+                            {typeof rawValue === 'undefined'
+                                ? ''
+                                : `(${typeof rawValue}) ${inspect(rawValue)}`}
+                        </Text1>
+
+                        <Text1 regular>
+                            onChangeValue:{' '}
+                            {typeof value === 'undefined' ? '' : `(${typeof value}) ${inspect(value)}`}
+                        </Text1>
+                    </Stack>
+                </Stack>
+            </Box>
+        </ResponsiveLayout>
+    );
+};
+
+Controlled.storyName = 'controlled';
+Controlled.args = {
+    initialValue: '13/21',
+    ...defaultBaseArgs,
+};
+
+interface CreditCardExpirationFieldUncontrolledArgs extends CreditCardExpirationFieldBaseArgs {
+    defaultValue: string;
+}
+
+export const Uncontrolled: StoryComponent<CreditCardExpirationFieldUncontrolledArgs> = ({
+    inverse,
+    defaultValue,
+    ...rest
+}) => {
+    const [rawValue, setRawValue] = React.useState<any>(undefined);
+    const [value, setValue] = React.useState<any>(undefined);
+
+    return (
+        <ResponsiveLayout isInverse={inverse} fullWidth>
+            <Box padding={16}>
+                <Stack space={16}>
+                    <div data-testid="credit-card-expiration-field-wrapper">
+                        <CreditCardExpirationField
+                            defaultValue={defaultValue}
+                            onChange={(e) => setRawValue(e.target.value)}
+                            onChangeValue={(value) => setValue(value)}
+                            name="creditCardExpiration"
+                            autoComplete="off"
+                            data-testid="credit-card-expiration-field"
+                            {...rest}
+                        />
+                    </div>
+                    <Stack space={8}>
+                        <Text1 regular>
+                            onChange:{' '}
+                            {typeof rawValue === 'undefined'
+                                ? ''
+                                : `(${typeof rawValue}) ${inspect(rawValue)}`}
+                        </Text1>
+
+                        <Text1 regular>
+                            onChangeValue:{' '}
+                            {typeof value === 'undefined' ? '' : `(${typeof value}) ${inspect(value)}`}
+                        </Text1>
+                    </Stack>
+                </Stack>
+            </Box>
+        </ResponsiveLayout>
+    );
+};
+
+Uncontrolled.storyName = 'uncontrolled';
+Uncontrolled.args = {
+    defaultValue: '13/21',
```

Comment:
well, if you want the page to load with a particular value already written in the field, how would you do it otherwise? For example, it's comfortable for screenshot/acceptance tests.